frontend/lib/analytics/google-analytics.tsx

```diff
@@ -146,19 +146,6 @@ export interface GoogleAnalyticsAPI {
     eventAction: "toggle"
   ): void;
 
-  /**
-   * A custom event for when the user toggles a dropdown.
-   *
-   * @param name The name of the dropdown that was toggled.
-   */
-  (
-    cmd: "send",
-    hitType: "event",
-    eventCategory: "dropdown",
-    eventAction: "toggle",
-    name: string
-  ): void;
-
```

Comment:
This was never really being used, since we only ever had the developer dropdown, which is only used for local development.  It doesn't really seem particularly helpful either, so I just removed it.